opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/InteroperabilityTest.java

```diff
@@ -422,42 +419,49 @@ void testSupportedMetricsExportedCorrectly() {
     assertThat(metricData.size()).isEqualTo(4);
 
     MetricData metric = metricData.get(0);
-    assertThat(metric.getName()).isEqualTo("double_gauge");
-    assertThat(metric.getDescription()).isEqualTo("double gauge");
-    assertThat(metric.getUnit()).isEqualTo("ms");
-    assertThat(metric.getType()).isEqualTo(MetricDataType.DOUBLE_GAUGE);
-    assertThat(metric.getDoubleGaugeData().getPoints().size()).isEqualTo(1);
-    PointData pointData = metric.getDoubleGaugeData().getPoints().iterator().next();
-    assertThat(((DoublePointData) pointData).getValue()).isEqualTo(60);
-    assertThat(pointData.getLabels().size()).isEqualTo(0);
-
+    assertThat(metric)
+        .hasName("double_gauge")
+        .hasDescription("double gauge")
+        .hasUnit("ms")
+        .hasDoubleGauge()
+        .points()
```

Comment:
Tend to like not switching gears in an assertion (i.e., allowing a further assertion on the metric after this one) with something like `pointsSatisfyExactly(...)`. One reason is for the indentation to match the structure. I guess it's mostly just personal preference though so don't mind too much

Reply:
I hear you. I chose the methods to match existing patterns in Assertj, particularly around Iterables/Collections.  

The TL;DR: is this matches Assertj even if I'm more of a hamcrest fan myself (primarily for composability).

Want to call out two things:

1. The goal of this library is folks STOP using exact ordering where it's not important (you can choose when, but prefer `inAnyOrder` if you don't know whether order matters).   If you look at examples there's a lot of `.points().satisfiesExactlyInAnyOrder`.
2. There's a lot of Iterable matches I don't want to do a combinatorial explosion on providing, similar for Attributes.   I could take an approach here (similar to what I did for Attribtues) where you can go into a `.points` *or* use a `hasPoints` that implicitly uses "containsInAnyOrder".